### ISSUE TITLE
Problems with New Format

### ISSUE THREAD

**Issue by alexsimon96** [meyerjo/deepdrive_dataset_tfrecord#4]
Good night

I was trying to use your application to transform from Berkeley Dataset to TFRecord. The issue is that the program expect more than one json file:

`Traceback (most recent call last):
  File "create_tfrecord.py", line 51, in <module>
    daytime_type=FLAGS.daytime, classes=FLAGS.classes
  File "/home/alejandro/deepdrive_dataset_tfrecord/deepdrive_dataset/deepdrive_dataset_writer.py", line 426, in write_tfrecord
    picture_id, full_labels_path=full_labels_path)
  File "/home/alejandro/deepdrive_dataset_tfrecord/deepdrive_dataset/deepdrive_dataset_writer.py", line 275, in get_annotation
    full_labels_path, picture_id + '.json'), 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/alejandro/.deepdrive/labels/bdd100k/labels/100k/3cffac71-5df609ef.json'`

As you can see, the problem is that i'm trying to work with the New Format of Berkely Dataset, so because of that I have only one .json file in the labels carpet. Indeed I have TWO .json files, one for training labels and another for validation labels.

I have been trying to force your code to use the functions that refer to the new version, but honestly It have been a mess, I have got a lot of errors.

To sum up, I would be very thankful if you can explain me how to use your application with the New Format of Berkeley Dataset. I have seen that you have developped functions for the new version, but i don't know how to use then.

Thank you for all in advance.
